### ISSUE TITLE
[CST] Small language tweak for 5103 document request page

### ISSUE THREAD

**Issue by skylerschain** [department-of-veterans-affairs/va.gov-team#89666]
## Background Context
Small tweak to sentence on 5103 document request page

### New Feature
- Why are we doing this work?
- Epic Link

## Outcome, Success Measure, KPI(S), and Tracking Link
- Add here

## Design
[FIGMA](https://www.figma.com/design/BUdMIxAiIutG12rZxZ0cg1/Claim-Letters-Navigation-%2B-5103-Alert-Updates?node-id=2012-1626&t=jXTE8OnryMiAm2U3-4)

We want to update the text in this paragraph here:
<img width="1120" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK2tlQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--96e258516f5b3cbfbd9a9969a63646b635fe7657/Screenshot%202024-07-30%20at%205.41.08%E2%80%AFPM.png" alt="Screenshot 2024-07-30 at 5.41.08 PM.png" />

BEFORE: "You don’t need to do anything on this page. We’ll wait until {date}, to move your claim to the next step." 

AFTER: "We'll wait for you to send more evidence until {date}. Then we'll move your claim to the next step. Or you can tell us sooner that you don't have more evidence to submit."

### Enablement team (if needed)
@jstrothman 

## Engineering 
- Add here

## Out of scope
- Add here

## Open questions
- Add here

## Tasks
- [ ] _Change text for 5103 document request page as described above in the Design section_

## Definition of Done
- [ ] Meets acceptance criteria
- [ ] Passed E2E testing (90% coverage)
- [ ] Passed unit testing (90% coverage)
- [ ] Passed integration testing (if applicable)
- [ ] Code reviewed (internal)
- [ ] Submitted to staging
- [ ] Team approved production verification process
- [ ] Design performs design QA and verifies the implementation matches the design spec
- [ ] Accessibility specialist performs accessibility review (in code or design)
- [ ] Engineering identifies staging users required to test and shares account and credentials with design and product
- [ ] Product performs functional QA and verifies acceptance criteria was met

## Acceptance Criteria
- [ ] _Text specified above has been updated_
